config/wpcalypso.json

```diff
@@ -49,6 +49,7 @@
 		"manage/import/medium": true,
 		"manage/jetpack": true,
 		"manage/jetpack-plans": true,
+		"manage/media": true,
```

Comment:
Let's also turn this off for wpcalypso/dev, since we'll be focusing on other things for a while.